source/elements/oneDAL/source/notations.rst

```diff
@@ -4,4 +4,63 @@
 Mathematical Notations
 ======================
 
+.. list-table::
+   :widths: 15 85
+   :header-rows: 1
 
+   * - Notation
+     - Definition
+
+   * - :math:`n`
+     - Number of :term:`feature vectors <Feature vector>` in a :term:`training
+       set <Training set>`.
+
+   * - :math:`m`
+     - Number of feature vectors in a :term:`inference set <Inference set>`.
```

Comment:
Why do you want to introduce one letter for training set (`n`) and another for inference set (`m`)? I understand that we need to distinguish the number of observations for training and inference when we describe some algorithm. I'd better introduce one term `n` for a number of feature vectors in a **dataset** without specifying what kind of dataset it is. We can refer to a number of observations in a inference dataset as `n_infer`, for example. It is inconvenient to have two letters for the same thing in case of data management - which letter should I use when describing a shape of a table, which can be used on the both stages?

Reply:
And more generally, this is a number of observations, not a feature vectors only

Reply:
We can reserve both `n` and `m` for a general dataset, and mention `m` will be used if we need to distinguish two datasets.

Reply:
Regarding `n_train`, to me it seems to wordy. In fact, it should be written in latext this way `$n_{\mathrm{train}}$`, which turns compact formula into unparsable sequence of braces:

Before `X' = \{ x_1', \ldots, x_m' \}`
 
Ater `X' = \{ x_1', \ldots, {x'}_{n_{\mathrm{train}}}\}`

Moreover, I don't very like multiple levels of indices in formulas:

![](http://latex2png.com/pngs/0a854bf711e45670b5078b0966bfbf94.png)